pulseapi/settings.py

```diff
@@ -25,10 +25,12 @@
     DEBUG=(bool, False),
     USE_S3=(bool, False),
     SSL_PROTECTION=(bool, False),
-    CORS_REGEX_WHITELIST=(tuple, ()),
+    CORS_ORIGIN_REGEX_WHITELIST=(tuple, ()),
```

Comment:
how come this is `(tuple, ())` while the others are `(list, [])`? based on the documentation for the cors header middleware, we can set them all to be `(tuple, ())` without issue (unless I missed something)

Reply:
It's probably me who misread the doc at some point: we can simply use list for the regex too.